source/PowerUp.js

```diff
@@ -0,0 +1,43 @@
+var Pixi = require("pixi.js")
+var Afloop = require("afloop")
+var Keyb = require("keyb")
+
+import Junkership from "./Junkership.js"
+import Reference from "./Reference.js"
+import Score from "./Score.js"
+import GameContainer from "./GameContainer.js"
+import Projectile from "./Projectile.js"
+
+export default function powerUp(curPower) {
+
+    this.curPower = curPower
+    this.bulletSpeed = 80
+    this.fireRate = 1
+    this.projectileType = "bullet"
+
+    this.peaShoota = peaShoota
+    this.triShoot = triShoot
+
+    function peaShoota(curXpos,curYpos,curShip,wOffset,hOffset) {
+        game.addChild(new Projectile(
+                curXpos + wOffset,
+                curYpos + hOffset,
+                curShip, this.fireRate, this.bulletSpeed, this.projectileType))
+    }
```

Comment:
Finishing up from the earlier comments, if you just pass this function `curShip`, you should be able to then do

 ```
   function peaShoota(curShip) {
         game.addChild(new Projectile(
                 curShip.position.x + curship.width,
                 curShip.position.y + curShip.height,
                 curShip, this.fireRate, this.bulletSpeed, this.projectileType))
     }
```

Which saves us a bit of throwing stuff around. The Junkership is already coming with its attributes, so no need to pass them twice, y'know?